packages/fxa-payments-server/server/lib/server.js

```diff
@@ -116,7 +143,7 @@ module.exports = () => {
           userReq /*, userRes*/
         ) {
           const contentType = proxyRes.headers['content-type'];
-          if (!contentType || !contentType.startsWith('text/html')) {
+          if (! contentType || ! contentType.startsWith('text/html')) {
```

Comment:
This was some linter complaint, although I believe we've gotten a fix in now so I could change this back🤷‍♀️

Reply:
Huh, yeah, I thought the style rules were consolidated.